src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherForm.java

```diff
@@ -38,6 +49,26 @@ public String getTripID() {
 		return StringUtils.EMPTY;
 	}
 	
+	public boolean isCancelPDPPaymentsActionAvailable() {
```

Comment:
There's some odd indentation in this new method. I would suggest changing this method's tabs into spaces to clean up the formatting.